src/doc/dev_manual/CI.rst

```diff
@@ -0,0 +1,29 @@
+Docker Containers For CI Testing
+=================================
+
+We use Azure Pipelines for CI testing Visit's Pull Requests.
+
+* `Ascent Azure DevOps Space <https://dev.azure.com/visit-dav/VisIt/>`_
+
+
+To speed up our CI testing we use Docker containers with pre-built third party libraries. These containers leverage our ``build_visit`` third party build  process. The Docker files and build scripts used to create these containers are in ``scripts/ci/docker``. To update the containers (assuming you have Docker installed):
+
+ * Run ``python build_docker_visit_ci.py`` to build and tag new version of our TPL container.
+
+  This tag will include today's day and a short substring of the current git hash.
+  Example Tag: ``visitdav/visit-ci-develop:2020-08-24-shac49c21``
+
+ * Run ``docker push <container-name>`` to push the container image to `VisIt's DockerHub Registry <https://hub.docker.com/orgs/visitdav>`_.
+
+  You will need to be logged into DockerHub to successfully push, the process may ask for your DockerHub username and password. Example Push Command: ``docker push visitdav/visit-ci-develop:2020-08-24-shac49c21``
```

Comment:
Make `DockerHub` link to the actually place you need to be logged into

Reply:
You log in via the command line to push -- if you run docker push it will ask for your credentials

Reply:
Ok, so maybe add...

If you do not already have a DockerHub account, go [here](https://hub.docker.com/signup) and sign up for one (assuming that is correct link).

Reply:
good suggestion, I'll add info about the fact you need to be in the visitdav org as well.